### ISSUE TITLE
Accessibility: 

### ISSUE THREAD

**Issue by james95-developer** [jupyterhub/outreachy#51]
### What page is this for?

https://jupyter.org/governance/overview.html

### WAVE accessibility report

![overview](https://user-images.githubusercontent.com/97939107/195828248-b39b9cb2-4f3c-4071-9fd6-bbc97094b0cf.PNG)

In reviewing the above page I came a cross the following
1. form label is present, but does not contain any content.
A `<label>` element that is associated to a form control but does not contain text will not present any information about the form control to the user.
```
<label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
<span class="headerbtn__icon-container">
<i class="fas fa-bars"></i>
</span>
</label>
```
**Suggested changes**
Ensure that the form label contains text that describes the function of the associated form control. Labels are not required for image, submit, reset, button, or hidden form controls. If a label is not necessary visually, a descriptive title attribute may be added to the form control.
2.The language of the document is not identified or a `lang` attribute value is invalid. `<base href="https://jupyter.org/governance/overview.html">`
 **Suggested changes**
 Identify the document language using the <html lang> attribute with a valid value (e.g., <html lang="en">). Ensure that all lang attribute values are valid.
3. A form control has more than one label associated with it. `<input type="checkbox" class="sidebar-toggle" name="__page- 
toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">`
**Suggested changes**
Ensure that at most one label element is associated to the form control. If multiple form labels are necessary, use aria-labelledby or to wrap the input in the label

4. Suspicious link text. Link text contains extraneous text or may not make sense out of context.
```
<p>
More details on these can be found
<a class="reference internal" href="standing_committees_and_working_groups.html">
<span class="doc std std-doc">
here
</span>
</a>
```
 **Suggested changes**
Where appropriate, reword the link text so that it is more descriptive of its destination when read out of context. Remove any extraneous text (such as "click here").

### The HTML element to be changed

```html
**Suggested elements to be changed**
1.One way to solve this is with visually hidden text, which assistive technology can access but is not visible on the screen visually. Using the visually-hidden class.
<span class="theme-switch-wrapper">
     <label for="__page-toc" class="headerbtn headerbtn-page-toc">
<span class="headerbtn__icon-container">
<i class="fas fa-list"></i>
</span>
<span class="visually-hidden">Label for the input</span>
</label>
</span>
2.Adding lang attribute
<base href="https://jupyter.org/governance/overview.html">`
3.One way to fix this other than changing IDs is to wrap the input in the label.
<label>
    <input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<input />
</label
4.Reword the link
<p>
More 
<a class="reference internal" href="standing_committees_and_working_groups.html">
<span class="doc std std-doc">
details on these can be found
</span>
</a>
```

### Your proposed HTML (or CSS), after change

```html
1. css for the visuallly hidden class for the label will look like this 
.visually-hidden { 
    border: 0;
    padding: 0;
    margin: 0;
    position: absolute !important;
    height: 1px; 
    width: 1px;
    overflow: hidden;
    clip: rect(1px 1px 1px 1px); 
    clip: rect(1px, 1px, 1px, 1px); 
    clip-path: inset(50%); 
    white-space: nowrap; 
}
```

**Comment by minrk** [comment]
Thanks for these suggestions! I wonder if we can find some accessible examples of the collapsible header element. Presumably that element should be entirely omitted from the screen reader context, since the header nav should just be accessible for a reader. Or is that not correct?

**Comment by james95-developer** [comment]
@minrk Thanks let me have a look at it in more details

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
